re2.asl

```diff
@@ -810,17 +837,25 @@ split
         }
     }
 
+    // Health lists are [current, max]
+    int[] g1HP = vars.GetBossHealth("G1");
+    int[] g2HP = vars.GetBossHealth("G2");
+    int[] g3HP = vars.GetBossHealth("G3");
+    int[] g4HP = vars.GetBossHealth("G4");
+    int[] g5HP = vars.GetBossHealth("G5");
+    int[] tyrantHP = vars.GetBossHealth("Tyrant");

```

Comment:
These could probably be moved to inside the normal checks so that we aren't looping on every single split check
```asl
if (normalCondition)
{
    int[] hp = vars.GetBossHealth("G2");
    if (hp[0] == .... && hp[1] == ....)
        return LogAndSplit("....."):
}
```